ibcdfo_pypkg/setup.py

```diff
@@ -1,47 +1,50 @@
 import codecs
 from pathlib import Path
 
-from setuptools import find_packages, setup
+from setuptools import setup
 
-_FILE_PATH = Path(__file__).parent
+_PKG_ROOT = Path(__file__).resolve().parent
 
 
 def readme_md():
-    fname = _FILE_PATH.joinpath("README.md")
+    fname = _PKG_ROOT.joinpath("README.md")
     with codecs.open(fname, encoding="utf8") as fptr:
         return fptr.read()
 
 
 def version():
-    fname = _FILE_PATH.joinpath("VERSION")
+    fname = _PKG_ROOT.joinpath("VERSION")
     with open(fname, "r") as fptr:
         return fptr.read().strip()
 
 
-# wheel must be installed before installing this package so that the LICENSE
-# file is actually installed as part of the source distribution when it is
-# installed with pip.  This implies that the LICENSE is in the sdist tarball for
-# those who access the code outside of pip.
-python_requires = ">=3.7"
-install_requires = ["wheel", "numpy>=1.16.5", "scipy>=1.6"]
-# TODO: Can we integrate testing using tox or other solution?
-# test_requires = ["oct2py"]
+# Changes made to python_requires should be propagated to all tox.ini and all
+# GitHub Action config files.
+python_requires = ">=3.8"
+code_requires = ["numpy", "scipy"]
+test_requires = ["ipdb"]  # "BenDFO" is required, but not yet installable
+install_requires = code_requires + test_requires
 
-packages = find_packages(include=["ibcdfo", "ibcdfo.pounders"])
-# TODO: This is a hack because these are python files rather than package data,
-# which could be something like data files needed by tests.  I believe that this
-# is unnecessary if test file organization follows the pattern
-#                         test/test*.py
-package_data = {"ibcdfo.pounders": ["tests/*/*.py"]}
+package_data = {
+    "IBCDFO": [
+        "pounders/tests/regression_tests/benchmark_resutls/*.txt",
```

Comment:
results is misspelled.  Also, the benchmark results folder is not in the regression tests folder.

Reply:
This has been fixed